src/main/java/hudson/plugins/parameterizedtrigger/BlockableBuildTriggerConfig.java

```diff
@@ -7,12 +7,10 @@
 import hudson.model.AbstractProject;
 import hudson.model.Action;
 import hudson.model.BuildListener;
-import hudson.model.Cause.UpstreamCause;
-import hudson.model.Hudson;
 import hudson.model.Job;
 import hudson.model.Node;
 import hudson.model.Run;
-import jenkins.model.ParameterizedJobMixIn;
```

Comment:
Was this just not used anywhere?

Reply:
Exactly. It was introduced in commit e5a2f50d540e0262739c9c45086563782634dff0 but never used.